libursa/src/bn/openssl.rs

```diff
@@ -540,6 +580,8 @@ impl BigNumber {
         Ok(qr)
     }
 
+    // Question: Why does this need to be a Result? When is creating a BigNumber same as another
+    // BigNumber not possible given sufficient memory?
```

Comment:
Same comment as above

Reply:
OpenSSL checks the stack to make sure it can allocate the object. If it can't then it fails. I have never seen it fail so it's probably okay to change it to just return a BigNumber. My other reasoning is that GMP doesn't do this check and neither does the Rust Bignumber library that we are using as an alternative to OpenSSL.

Reply:
That kind of a change should be done as a separate PR as it touches a lot of code.